app/controllers/topics_controller.rb

```diff
@@ -61,6 +61,18 @@ def destroy
     end
   end
 
+  def upvote
+    @topic = Topic.find(params[:id])
+    @topic.votes.create
+    redirect_to(topics_path)
+  end
+
+  def downvote
+    @topic = Topic.find(params[:id])
+    @topic.votes.last.destroy
```

Comment:
:rotating_light: It's possible that `@topic.votes.last` will return `nil` if there are currently 0 votes in the system. (think of `nil` like JavaScript's `null` – calling `array.last` on an empty array will return `nil`).

What are some ways to deal with this? Is it even a problem?